jit/v1/jit_v1.2.0-Beta-2.yaml

```diff
@@ -0,0 +1,1312 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      operationId: get-v1-events
+      x-stoplight:
+        id: vnvljyf8iq2y2
+      description: "Retrieves Operations events. The default sort order is by eventCreatedDateTime in descending order (DESC).\r\n\r\nIt is possible to combine queryParameters. When combing queryParameters the logical AND is used between them. The response list must fulfill all queryParameters. An example of this is when the following queryParameters are used:\r\n\r\n**eventCreatedDateTime:gte=2021-04-01T14:12:56+01:00 and vesselIMONumber=9321484**\r\n\r\nthen all events in the response list returned must be connected to vesselIMONumber 9321484 AND have an eventCreatedDateTime ≥ 2021-04-01T14:12:56+01:00\r\n\r\nNB: When combining queryParameters be aware that it is also possible to make combinations that are mutual contradicting! An example of this could be when the following queryParameters are used:\r\n\r\n**vesselIMONumber=9321484 and carrierServiceCode=FE1**\r\n\r\nif there are no Vessels with IMO number 9321484 on Carrier Service Code FE1. If this is the case then this response **will return an empty list!**\r\n\r\n"
+      parameters:
+        - schema:
+            type: string
+            example: 123e4567-e89b-12d3-a456-426614174000
+            maxLength: 100
+          in: query
+          name: transportCallID
+          description: 'ID uniquely identifying a transport call, to filter events by.  Specifying this filter will only return events related to this particular transportCallID.'
+        - schema:
+            type: string
+            example: '9321483'
+            maxLength: 7
+          in: query
+          name: vesselIMONumber
+          description: 'The identifier of vessel for which schedule details are published. Depending on schedule type, this may not be available yet.  Specifying this filter will only return events related to this particular vesselIMONumber.'
+        - schema:
+            type: string
+            example: 2103S
+            maxLength: 50
+          in: query
+          name: carrierVoyageNumber
+          deprecated: true
+          description: 'Filter on the vessel operator-specific identifier of the Voyage.  Specifying this filter will only return events related to this particular carrierVoyageNumber.  Deprecated: Use exportVoyageNumber instead.'
+        - schema:
+            type: string
+            example: 2103S
+            maxLength: 50
+          in: query
+          name: exportVoyageNumber
+          description: 'Filter on the vessel operator-specific identifier of the export Voyage.  Specifying this filter will only return events related to this particular exportVoyageNumber.  Deprecated: Use carrierExportVoyageNumber instead.'
+          deprecated: true
+        - schema:
```

Comment:
In the "current" JIT 1.2 there is a `carrierExportVoyageNumber` query parameter - I cannot seem to find it here:
![image](https://github.com/dcsaorg/DCSA-OpenAPI/assets/3489388/20109510-be4b-4dfa-b9c2-771105d52115)